### ISSUE TITLE
Conditional Logic bug with "Toggle Visibility" action "show/hide"

### ISSUE THREAD

**Issue by akno77** [dachcom-digital/js-pimcore-formbuilder#6]
I have the following situation:

- I have a radio box element with option yes/no and and a input field
- The input field should only be visible if the option "yes" was selected
- I added a Conditional Logic on the radio element saying if the value is not "yes" the action is "Toggle Visibility" of the input field to "hide"
- The input field is hidden and if I select "yes" the field is displayed. But if I select "no" again both fields (the radio and input) are hidden!

**Comment by akno77** [comment]
I tried now to use the "Toggle Availability" action instead, but it seems it has exactly the same problem...

**Comment by akno77** [comment]
I found the reason why it doesn't work for me. I named the radio field "certificatesAvailable" and the text input field "certificates". When the action is executed there is a query selector for all fields that start with "certificates", so the "certificatesAvailable" is returned accidentally, too.

It is caused in **src/js/components/conditionalLogic.js** on line 133:
`let field = this.form.querySelectorAll('[name*="' + fieldName + '"]');`

I think the selector should not be a wildcard...
